Skylab/Custom elements/CustomView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CustomView.swift
+//  Skylab
+//
+//  Created by Artem Tkachenko on 22.02.2023.
+//
+
+import UIKit
+
+@IBDesignable class CustomView: UIView {
+    
```

Comment:
BTW according to the US, an extension should be created, but now I think what you've done is probably a better idea  👍